source/Weapon.cpp

```diff
@@ -79,6 +79,7 @@ void Weapon::LoadWeapon(const DataNode &node)
 		}
 		else
 		{
+<<<<<<< HEAD
```

Comment:
This shouldn't be here :)

Reply:
Also, there are perhaps other places now that need WeightedVelocity:

https://github.com/endless-sky/endless-sky/pull/1894/files#diff-80825e2b1f90be6b565f227d9ddb18ccR1926
https://github.com/endless-sky/endless-sky/pull/1894/files#diff-80825e2b1f90be6b565f227d9ddb18ccR2025
https://github.com/endless-sky/endless-sky/pull/1894/files#diff-80825e2b1f90be6b565f227d9ddb18ccR2264

There may be more.